lib/config/presets/internal/monorepo.ts

```diff
@@ -265,6 +265,7 @@ const repoGroups = {
   'ember-decorators': 'https://github.com/ember-decorators/ember-decorators',
   emojibase: 'https://github.com/milesj/emojibase',
   emotion: 'https://github.com/emotion-js/emotion',
+  eslint: 'https://github.com/eslint/eslint',
```

Comment:
Do any of our existing rules already group eslint packages and would conflict with this? If so then we need to decide which changes to make

Reply:
we've some linter grouping rules

Reply:
Those are opt-in I think

Reply:
@fuxingloh how many packages are published from github.com/eslint/eslint other than eslint itself?

Reply:
Seems not many: https://github.com/eslint/eslint/tree/main/packages